cmd/manager/main.go

```diff
@@ -59,13 +57,17 @@ type clusterConfig struct {
 	network clusternetwork.ClusterNetworkConfig
 }
 
-func printVersion() {
-	log.Info("operator", "version", version.Version)
-	log.Info("go", "version", runtime.Version(), "os", runtime.GOOS, "arch", runtime.GOARCH)
-	log.Info("operator-sdk", "version", sdkVersion.Version)
-}
-
 func main() {
+	// add version subcommand to query the operator version
+	if len(os.Args) > 1 {
+		switch os.Args[1] {
+		case "version":
+			fmt.Printf("%s version: %q, go version: %q\n", os.Args[0], version.Get(),
+				version.GoVersion)
+			os.Exit(0)
```

Comment:
What about default? Shouldn't we error out?

Reply:
Do we want to have `long version` like `windows-machine-config-operator` as `args[0]`?

Reply:
args[0] will need some additional changes, which is not a part of this PR.

as far as default option goes, we'll be having flags for zap logger/etc. if we add default, it'll cause errors. 
command line option is just a nice to have for now. imo args[0] will suffice for now.